### ISSUE TITLE
[Mercure][Feature] Configure automatic mercure updates

### ISSUE THREAD

**Issue by stephanvierkant** [api-platform/core#3785]
As far as I can see, API Platform sends updates on insert/update/delete (`\ApiPlatform\Core\Bridge\Doctrine\EventListener\PublishMercureUpdatesListener::onFlush`) automatically. This causes performance problems if many entities are updated/inserted/deleted at once. In my case, I'm not listening to inserts and deletions, so it doesn't make sense to send the updates in the first place.

It would be great if some kind of configuration would be possible.

**Comment by soyuka** [comment]
This is true only with `mercure=true` on an entity right? I'd say that you should definitely override our `PublishMercureUpdatesListener` to have all the control you need. I've done so on few projects already.
